### PR TITLE
Fixed ZipCode Controller and Tests

### PR DESCRIPTION
**Pull request by tim-choi**
Fixed issue where some variables were not right.

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/ZipCodeController.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import io.swagger.v3.oas.annotations.Parameter;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
-@Tag(name="Country Code info from http://api.zippopotam.us/us/{zipcode}")
+@Tag(name="Zip Code info from http://api.zippopotam.us/us/{zipcode}")
```
Comment (ericpretzel) [inline src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/ZipCodeController.java]: Can we change this link to just http://api.zippopotam.us/ to make it consistent with the example site

**Review by ericpretzel — CHANGES_REQUESTED** [review]
LGTM other than the one thing I requested

**Review by KevinChen0527 — CHANGES_REQUESTED** [review]
Could you change line 35 example to a zip code

**Review by KevinChen0527 — APPROVED** [review]
lgtm

**Review by ericpretzel — APPROVED** [review]

**Pull request merged**